gallery/src/components/NxCheckbox/NxCheckboxPage.tsx

```diff
@@ -94,6 +94,21 @@ const NxCheckboxPage = () =>
               <NxCode>label</NxCode> element.
             </NxTable.Cell>
           </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell>checkboxAttributes</NxTable.Cell>
+            <NxTable.Cell>
+              <NxTextLink external href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox">
+                HTML Input Checkbox Attributes
+              </NxTextLink>
+            </NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>
+              NxCheckbox also supports any html attribute that's normally supported by the
+              <NxCode>input</NxCode> element except for <NxCode>id</NxCode>, <NxCode>disabled</NxCode>,
```

Comment:
The wording here makes it sound like `NxCheckbox` doesn't support externally setting `disabled`.  It does, just not through this prop.  I'd rephrase this description to be less copy/pasted from our usual native attr docs and clearer for the situation at hand.

Reply:
> The wording here makes it sound like `NxCheckbox` doesn't support externally setting `disabled`. It does, just not through this prop. I'd rephrase this description to be less copy/pasted from our usual native attr docs and clearer for the situation at hand.

Thanks I tried to rephrase it here: 4c796beb23b05878976a76dfdcf72610774430aa